.github/workflows/weekly-release.yml

```diff
@@ -1,6 +1,7 @@
 name: Weekly Release
 
 on:
+  workflow_dispatch:
```

Comment:
Cool! Perhaps we also could have a version number input, in case we decide to release new minor version instead of operating over patch versioning?

Reply:
That's a good idea, but more complicated... Let's do that, but apart.

Reply:
I'll create a ticket